### PR TITLE
chore: iterate upload code rearranged for clear readability

### PR DESCRIPTION
**Pull request by JasonPaulGithub**
## Description

reviewpad:summary 

Iterate upload reads a lot easier from the top down. I would apply test docs but this function may soon see further adaptations for generic use by its two parent functions, but this will do for now.

### CODE REVIEW

File: sn_cli/src/subcommands/files/iterative_uploader.rs [hunk 1]
```diff
@@ -104,7 +110,8 @@ fn spawn_progress_handler(
 ) -> JoinHandle<Result<(), Error>> {
     tokio::spawn(async move {
         let mut upload_terminated_with_error = false;
-        // The loop is guaranteed to end, as the channel will be closed when the upload completes or errors out.
+        // The loop is guaranteed to end, as the channel will be
```
Comment (jacderida) [inline sn_cli/src/subcommands/files/iterative_uploader.rs:114]: Really minor issue, but your line length seems a bit small for these comments. I think `rustfmt` uses 100 as a line length, which is a bit more modern than typical values like 72 or 80. I think that's also a decent line length for comments and doc comments.

Comment (JasonPaulGithub) [reply sn_cli/src/subcommands/files/iterative_uploader.rs:114]: resolved

File: sn_cli/src/subcommands/files/iterative_uploader.rs [hunk 2]
```diff
@@ -95,7 +101,7 @@ impl IterativeUploader {
 
 ///////////////// Associative Functions /////////////////
 
-fn spawn_progress_handler(
+fn spawn_file_upload_events(
```
Comment (RolandSherwin) [inline sn_cli/src/subcommands/files/iterative_uploader.rs]: This function handles the file upload events that we receive, so `handler` seems more apt here?

Comment (JasonPaulGithub) [reply sn_cli/src/subcommands/files/iterative_uploader.rs]: i try to avoid descriptions like 'handler', its too generic

Comment (JasonPaulGithub) [reply sn_cli/src/subcommands/files/iterative_uploader.rs]: can we just call it **spawn_file_upload_events_handler**
the name gets a little longer but this PR has been up for days and I just want to get this through

Comment (RolandSherwin) [reply sn_cli/src/subcommands/files/iterative_uploader.rs]: Yes! I feel that is more descriptive!

**Review by RolandSherwin — APPROVED** [review]

File: sn_cli/src/subcommands/files/iterative_uploader.rs (hunk 2 above)
Comment (JasonPaulGithub) [reply sn_cli/src/subcommands/files/iterative_uploader.rs]: changes made. cheers roland

**Pull request merged**